frontend/app/equity/dividend_computations/[id]/page.tsx

```
@@ -0,0 +1,328 @@
+
+function DividendComputationDetailContent() {
+  const params = useParams();
+  const [computation, setComputation] = useState(mockComputation);
+  const [isLoading, setIsLoading] = useState(false);
+  const isConfirmed = !!computation.confirmed_at;
+
+  useEffect(() => {
+    // Load computation data from localStorage for demo
+    const computationId = parseInt(params.id as string);
+    const demoComputations = JSON.parse(localStorage.getItem('demoComputations') || '[]');
+    const foundComputation = demoComputations.find((comp: any) => comp.id === computationId);
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)